app/jobs/submit_catalog_rest_job.rb

```diff
@@ -5,14 +5,22 @@ class SubmitCatalogRestJob < ActiveJob::Base
   # @param base_url [String] the base url to provide to the user in their order to check on for status updates
   def perform(order, base_url)
     request_url = "#{base_url}/data/retrieve/#{order.retrieval_collection.retrieval.to_param}"
+    shapefile = nil
+
+    if order.search_params.key?('sf') and not order.search_params['sf'].blank?
```

Comment:
I think just to match the rest of the project we should change `and not` to be `&& !`